R/resample.R

```diff
@@ -110,28 +110,40 @@ resample = function(learner, task, resampling, measures, weights = NULL, models
   time2 = Sys.time()
   runtime = as.numeric(difftime(time2, time1, units = "secs"))
   addClasses(
-    mergeResampleResult(learner, task, iter.results, measures, rin, models, extract, keep.pred, show.info, runtime),
+    mergeResampleResult(learner$id, task, iter.results, measures, rin, models, extract, keep.pred, show.info, runtime),
```

Comment:
This seems like a base change. @berndbischl needs to look into this and see if this could cause any problems or backwards incompatibilities.